File: CS/DXApplication1/DataAccess/AdventureWorksLTContext.cs
namespace DXApplication {
    using System;
    using System.Data.Entity;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Linq;
    using DXApplication.Data;

    public partial class AdventureWorksLTContext : DbContext {
        public AdventureWorksLTContext()
            : base(ConnectionSettings.GetConnectionString()) {
        }
        public virtual DbSet<Product> Products { get; set; }
    }
}

File: VB/DXApplication1/DataAccess/AdventureWorksLTContext.vb
Imports DXApplication.Data
Imports System.Linq
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Data.Entity
Imports System

Namespace DXApplication

	Partial Public Class AdventureWorksLTContext
		Inherits DbContext

		Public Sub New()
			MyBase.New(ConnectionSettings.GetConnectionString())
		End Sub
		Public Overridable Property Products() As DbSet(Of Product)
	End Class
End Namespace
